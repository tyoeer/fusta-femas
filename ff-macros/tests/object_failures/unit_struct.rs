use ff_macros::*;

#[derive(Object)]
struct Test;

fn main() {}