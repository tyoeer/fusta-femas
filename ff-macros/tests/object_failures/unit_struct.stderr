error: unit struct has no possible id field
 --> tests/object_failures/unit_struct.rs:4:1
  |
4 | struct Test;
  | ^^^^^^^^^^^^
